source/configuration_manual/authentication/auth_policy.rst

```diff
@@ -168,6 +168,46 @@ produces
 
 for this to work.
 
+List of fields
+--------------
+
+All fields supported by :ref:variables-auth can be used. In addition, you can use following fields:
+
+``hashed_password``
+  User's password hashed with password hash algorithm.
```

Comment:
I guess based on auth_policy_hash_* settings? Could have some pointer there. Or "with the configured password hash algorithm."

Reply:
it literally says "password hash algorithm" which is title few lines up =)

but i'll put a link to it.